android-configure

```diff
@@ -8,8 +8,6 @@ command -v python3.11 >/dev/null && exec python3.11 "$0" "$@"
 command -v python3.10 >/dev/null && exec python3.10 "$0" "$@"
 command -v python3.9 >/dev/null && exec python3.9 "$0" "$@"
 command -v python3.8 >/dev/null && exec python3.8 "$0" "$@"
-command -v python3.7 >/dev/null && exec python3.7 "$0" "$@"
```

Comment:
Where is Py3.12?

Reply:
IMO whether or not 3.12 is here is an issue for a different PR. This PR specifically removes up until 3.8.

FWIW 3.12 is used in the Node.js CI environment

Reply:
I think that's a different discussion thats out of scope of this pull-request.

Reply:
I agree that the discussion of 3.12 can be pushed to a separate change.

Reply:
OK.  I or someone else will revisit these files to complete the task.